addons/sys_core/fnc_updateExternalIntercomStatus.sqf

```diff
@@ -0,0 +1,45 @@
+/*
+ * Author: ACRE2Team
+ * SHORT DESCRIPTION
+ *
+ * Arguments:
+ * 0: Vehicle with intercom <OBJECT>
+ * 1: Unit using the intercom externally <OBJECT>
+ * 2: Type of action <NUMBER>
+ * 3: Unit giving the intercom telephone <OBJECT><OPTIONAL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [cursorTarget, player, 1] call acre_sys_core_updateExternalIntercomStatus
+ * [cursorTarget, infantryUnit, 1, player] call acre_sys_core_updateExternalIntercomStatus
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle", "_intercomUnit", "_action", "_givingUnit"];
+
+switch (_action) do {
+    case 0: {
+        // Stop using the intercom externally
+        [_vehicle, ["intercomUnit", nil, true]] remoteExecCall ["setVariable", _vehicle];
+        _intercomUnit setVariable ["vehicleIntercom", nil, true];
```

Comment:
`_intercomUnit setVariable ["vehicleIntercom", nil, true];` is already global so there is no need to use `remoteExecCall` I think using `setVariable` with the global flag (3rd parameter as true) instead of `remoteExecCall` is probably better. Do this for all instances in this function.